misk/src/main/kotlin/misk/backoff/ExponentialBackoff.kt

```diff
@@ -0,0 +1,71 @@
+package misk.backoff
+
+import java.time.Duration
+import java.util.concurrent.ThreadLocalRandom
+
+/**
+ * Performs exponential backoff with optional jitter. Durations are supplied as
```

Comment:
why an optional jitter? or at least make it the default

Reply:
What's a good default (given that we don't know any either the base or the max delay)?

Reply:
Maybe a default percentage?

Reply:
You ok if I deal with this in another PR? I have a higher priority branch that's using backoffs that I want to get in first.

Reply:
definitely. just a suggestion